src/libsync/CMakeLists.txt

```diff
@@ -15,6 +15,12 @@ if ( APPLE )
     )
 endif()
 
+if ( WIN32 )
+    list(APPEND OS_SPECIFIC_LINK_LIBRARIES
+        ws2_32
```

Comment:
Why?
Does the zsync library somehow directly use win sock? Can this be avoided? All network IO should be via Qt.

Reply:
It's because it needs `ntohs` and `ntohl` functions ...